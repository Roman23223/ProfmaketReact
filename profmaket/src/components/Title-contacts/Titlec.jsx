const Titlec = () => {
    return ( 
        <section className="section-works-title">

            <div className="works-title-container">
                <div className="title-text">
                    Наши контакты
                </div>
            </div>

     </section>  
     );
}
 
export default Titlec;