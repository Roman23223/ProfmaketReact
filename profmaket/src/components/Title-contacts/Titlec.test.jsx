import renderer from 'react-test-renderer';
import Titlec from './Titlec';

describe('Titlec component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Titlec />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});