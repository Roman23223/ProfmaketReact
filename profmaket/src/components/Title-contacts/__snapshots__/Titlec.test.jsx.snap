// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titlec component renders correctly 1`] = `
<section
  className="section-works-title"
>
  <div
    className="works-title-container"
  >
    <div
      className="title-text"
    >
      Наши контакты
    </div>
  </div>
</section>
`;
